manifests/install.pp

```diff
@@ -17,26 +17,20 @@
           source => "https://dl.fedoraproject.org/pub/epel/epel-release-latest-${::operatingsystemmajrelease}.noarch.rpm",
         }
       }
-
-      if defined(Package[$package_name]) {
-        Package[$package_name] {
-          require => Yum::Install['epel-release']
-        }
+      Package <| title == $package_name |> {
+        require => Yum::Install['epel-release']
       }
-    }
-
-    case $::operatingsystemmajrelease {
-      '12.04', '14.04', '16.04' : {
-        apt::source { 'ppa_collectd':
-          location => 'http://ppa.launchpad.net/collectd/collectd-5.5/ubuntu',
-          repos    => 'main',
-          key      => {
-            'id'     => '7543C08D555DC473B9270ACDAF7ECBB3476ACEB3',
-            'server' => 'keyserver.ubuntu.com',
-          },
-        }
+    } elsif ($::operatingsystemmajrelease == '12.04' or $::operatingsystemmajrelease == '14.04' or $::operatingsystemmajrelease == '16.04') {
```

Comment:
This looks better than my approach, but I would use `$::operatingsystemmajrelease =~ /('12.04'|'14.04'|'16.04')/` instead of these long line.